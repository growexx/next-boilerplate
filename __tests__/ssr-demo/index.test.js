import { render } from '@testing-library/react'

import SsrDemoPage from '@src/pages/ssr-demo'

describe('<SsrDemoPage />', () => {
  it('should match the snapshot', () => {
    const {
      container: { firstChild }
    } = render(<SsrDemoPage />)

    expect(firstChild).toMatchSnapshot()
  })
})
