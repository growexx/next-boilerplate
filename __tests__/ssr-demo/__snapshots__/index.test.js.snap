// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SsrDemoPage /> should match the snapshot 1`] = `
<section>
  SSR Demo
</section>
`;
